checkin/models.py

```diff
@@ -30,14 +30,18 @@ def __str__(self):
 
 
 class UserSkill(models.Model):
-    level_choices = (
-        (1, "Nybegynner"),
-        (2, "Viderekommen"),
-        (3, "Ekspert"),
+    BEGINNER = 1
+    EXPERIENCED = 2
+    EXPERT = 3
+
+    LEVEL_CHOICES = (
+        (BEGINNER, "Nybegynner"),
```

Comment:
It would be nice to have translations of these, probably fits the theme and changes in this PR to do it here.

Reply:
Yes, I agree 🙂